cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AuthorizationHandler.java

```diff
@@ -62,14 +78,17 @@
     this.enabled = conf.getBoolean(Constants.Security.Authorization.ENABLED);
   }
 
-  private void createLogEntry(HttpRequest httpRequest, AuthorizationRequest request,
+  private void createLogEntry(HttpRequest httpRequest, @Nullable AuthorizationRequest request,
                               HttpResponseStatus responseStatus) throws UnknownHostException {
-    String reqBody = String.format("[%s %s %s]", request.getPrincipal(), request.getEntity(), request.getActions());
+
     AuditLogEntry logEntry = new AuditLogEntry();
     logEntry.setUserName(Objects.firstNonNull(SecurityRequestContext.getUserId(), "-"));
     logEntry.setClientIP(InetAddress.getByName(Objects.firstNonNull(SecurityRequestContext.getUserIP(), "0.0.0.0")));
     logEntry.setRequestLine(httpRequest.getMethod(), httpRequest.getUri(), httpRequest.getProtocolVersion());
-    logEntry.setRequestBody(reqBody);
+    if (request != null) {
+      logEntry.setRequestBody(String.format("[%s %s %s]", request.getPrincipal(), request.getEntity(),
```

Comment:
How do these entries print? We should probably create a jira to make sure this doesn't print JSONs and stuff. Unrelated to this PR though.